### PR TITLE
feat: Change proposal body limit and strategies limit for turbo spaces

### PR DESCRIPTION
**Pull request by ChaituVR**
Change proposal body limit and strategies limit based on space type

### How to test:
- Use UI from https://github.com/snapshot-labs/snapshot/pull/4495/
- Add 10 strategies in space settings for turbo spaces
- Add 8 strategies in space settings for normal spaces
- Proposal body with 20000 characters for turbo spaces
- Proposal body with 10000 characters for normal spaces 
- Update proposal with different chars length

### CODE REVIEW

File: test/unit/writer/settings.test.ts [hunk 1]
```diff
@@ -72,6 +80,27 @@ describe('writer/settings', () => {
...
+        mockGetSpace.mockResolvedValueOnce({ ...spacesGetSpaceFixtures, turbo: true });
+        return expect(
+          verify(
+            editedInput({
+              strategies: randomStrategies(12)
+            })
+          )
+        ).rejects.toContain('wrong space format');
+      });
     });
```
Comment (ChaituVR) [inline test/unit/writer/settings.test.ts:104]: Every time we want to increase number of strategies limit in snapshot.js, we have to change the limits here 🙈can i change it to pick from schema?

Comment (wa0x6e) [reply test/unit/writer/settings.test.ts:104]: Yes, was not aware it was exported

Comment (ChaituVR) [reply test/unit/writer/settings.test.ts:104]: or should be fine maybe?

Comment (wa0x6e) [reply test/unit/writer/settings.test.ts:104]: Dynamic will be better

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**